app.py

```diff
@@ -182,6 +182,25 @@ def login_():
     def logout_():
         return logout()
 
+    @app.route('/push-log', methods=['POST'])
```

Comment:
issue: shall we protect this endpoint against public access?

Reply:
I thought about this but if we protect it we can't log issues risen before and while the auth process is performed.
We could consider it a tradeoff, I'm not totally sure how to go about this

Reply:
Endpoint is now private in latest revision